spec/unit/provider/package/yum_spec.rb

```diff
@@ -12,6 +12,40 @@
     expect(described_class.specificity).to be < 200
   end
 
+  describe "should have logical defaults" do
+    [2, 2018].each do |ver|
+      it "should be the default provider on Amazon Linux #{ver}" do
+        allow(Facter).to receive(:value).with(:operatingsystem).and_return('amazon')
+        allow(Facter).to receive(:value).with(:osfamily).and_return('redhat')
+        allow(Facter).to receive(:value).with(:operatingsystemmajrelease).and_return(ver)
+        expect(described_class).to be_default
+      end
+    end
+
+    Array(4..7).each do |ver|
+      it "should not be default for redhat #{ver}" do
```

Comment:
The test name is `should not be default`, but it's checking that it is the default. Should probably fix the test name.

Reply:
Fixed.